.rubocop_todo.yml

```diff
@@ -158,21 +32,21 @@ Metrics/ClassLength:
 Metrics/CyclomaticComplexity:
   Max: 17
 
-# Offense count: 3
+# Offense count: 2
 # Configuration parameters: AllowHeredoc, AllowURI, URISchemes, IgnoreCopDirectives, IgnoredPatterns.
 # URISchemes: http, https
 Metrics/LineLength:
-  Max: 218
+  Max: 214
 
-# Offense count: 17
+# Offense count: 11
 # Configuration parameters: CountComments, ExcludedMethods.
 Metrics/MethodLength:
-  Max: 130
+  Max: 22
 
-# Offense count: 4
+# Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 158
+  Max: 104
```

Comment:
Why did all these `Metrics` maximums change?

Reply:
Follow up: I've played around with this a little bit and can't find a definitive reason why these changed. A theory is that, as offense counts have gone down, max Metric values have as well as their worse offenders have been resolved/ignored. If these Metric cops become an issue in the future, we can just alter them in the `.rubocop.yml` file. Thus, this PR is ready to be moved forward.